	UVoidFunction is used independent of U_ENABLE_DYLOAD,
	so also declare it independently.

diff -ur icu-49.rc.orig/source/common/putilimp.h icu-49.rc/source/common/putilimp.h
--- icu-49.rc.orig/source/common/putilimp.h	2012-03-02 23:12:00.000000000 +0100
+++ icu-49.rc/source/common/putilimp.h	2012-03-03 18:19:56.000000000 +0100
@@ -513,7 +513,7 @@
 #  endif
 #endif
 
-#if U_ENABLE_DYLOAD
+#if U_ENABLE_DYLOAD || 1
 /*  Dynamic Library Functions */
 
 typedef void (UVoidFunction)(void);
